modules/thermal_hydraulics/test/tests/components/hs_coupler_2d3d/tests

```diff
@@ -11,10 +11,36 @@
   []
 
   [test]
-    type = Exodiff
-    input = hs_coupler_2d3d.i
-    exodiff = hs_coupler_2d3d_out.e
-    requirement = 'The system shall couple a 2D heat structure with a 3D heat structure.'
+    requirement = 'The system shall couple a 2D heat structure with a 3D heat structure'
+
+    [conduction_plus_radiation]
+      type = Exodiff
+      input = hs_coupler_2d3d.i
+      cli_args = "
+        Components/hs_coupler/include_radiation=true
+        Components/hs_coupler/emissivity_2d=0.4
+        Components/hs_coupler/emissivity_3d=0.6
+        Outputs/file_base=conduction_plus_radiation"
+      exodiff = conduction_plus_radiation.e
+      detail = 'using conduction and radiation.'
+    []
+    [conduction_only]
+      type = Exodiff
+      input = hs_coupler_2d3d.i
+      cli_args = 'Outputs/file_base=conduction_only'
+      exodiff = conduction_only.e
+      detail = 'using conduction only.'
+    []
+    [convection_only]
+      type = Exodiff
+      input = hs_coupler_2d3d.i
+      cli_args = "
+        Components/hs_coupler/gap_thermal_conductivity=0
+        Components/hs_coupler/gap_htc=100
+        Outputs/file_base=convection_only"
+      exodiff = convection_only.e
+      detail = 'using convection only.'
+    []
```

Comment:
It might be better to replace the first and third test with one that would include all 3 effects together.

Reply:
I decided on individual because it was more difficult to verify that convection is working correctly when seeing a sum of 3 components.